doc/dependencies.md

```diff
@@ -0,0 +1,18 @@
+# Dependencies
+
+These are the dependencies needed to build Zen on Linux.
+
+| Dependency | Tested version |
+| --- | --- |
+| [Make](https://www.gnu.org/software/make/) | [4.3] |
+| [Binutils](https://www.gnu.org/software/binutils/) | [2.38] |
+| [Curl](https://curl.se/) | [7.81] |
+| [Bzip2](https://sourceware.org/bzip2/) | [1.0.8] |
+| [GCC](https://gcc.gnu.org) | [11.2.0] |
+| [Patch](https://savannah.gnu.org/projects/patch/) | [2.7.6] |
+| [Perl](https://www.perl.org/) | [5.34.0] |
+| [Autoconf](https://www.gnu.org/software/autoconf/) | [2.71] |
+| [Automake](https://www.gnu.org/software/automake/) | [1.16.5] |
+| [Libtool](https://www.gnu.org/software/libtool/) | [2.4.6] |
+| [pkg-config](https://www.freedesktop.org/wiki/Software/pkg-config/) | [0.29.2] |
+| [hexdump](https://www.gnu.org/software/coreutils/) | [2.37.2] |
```

Comment:
According to `man hexdump`, "The hexdump command is part of the util-linux package which can be downloaded from Linux Kernel Archive <https://www.kernel.org/pub/linux/utils/util-linux/>"